### PR TITLE
chore: upgrade jest package versions

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: babel.config.json [hunk 1]
```diff
@@ -5,19 +5,13 @@
       {
         "targets": {
```
Comment (stbrody) [inline babel.config.json:6]: Changes to this file are due to copying what we have in the js-ceramic repo

File: jest.config.json [hunk 2]
```diff
@@ -1,18 +1,12 @@
+
 {
-  "moduleFileExtensions": ["js", "json", "ts"],
```
Comment (stbrody) [inline jest.config.json:2]: Changes to this file are due to copying what we have in the js-ceramic repo

**Comment by ukstv** [comment]
@stbrody I would like to take over this. Old Jest presents a pressing problem for me.

**Comment by stbrody** [comment]
> @stbrody I would like to take over this. Old Jest presents a pressing problem for me.

that would be great, wasn't sure when I was going to be able to get back to this anyway

**Comment by ukstv** [comment]
Sweet, closed this in favour of https://github.com/ceramicnetwork/ceramic-anchor-service/pull/827

**Pull request closed without merging**